cea/plots/thermal_networks/network_plot.html

```diff
@@ -0,0 +1,502 @@
+<div id="{{ hash }}" style="height: 100%; position: relative;">
+  <div id="layers-group-{{ hash }}" style="visibility: hidden;">
+    <span class="layer-toggle-{{ hash }}">
+      <label id='connected-{{ hash }}-label' class="map-plot-label">
+        <input id='connected-{{ hash }}' type='checkbox' name='layer-toggle-{{ hash }}' value='connected'>
+        Connected Buildings
+      </label>
+    </span>
+    <span class="layer-toggle-{{ hash }}">
+      <label class="map-plot-label">
+        <input id='disconnected-{{ hash }}' type='checkbox' name='layer-toggle-{{ hash }}' value='disconnected'>
+        Disconnected Buildings
+      </label>
+    </span>
+    <span class="layer-toggle-{{ hash }}">
+      <label class="map-plot-label">
+        <input id='district-{{ hash }}' type='checkbox' name='layer-toggle-{{ hash }}' value='district'>
+        District
+      </label>
+    </span>
+    <span class="layer-toggle-{{ hash }}">
+      <label id='network-{{ hash }}-label' class="map-plot-label">
+        <input id='network-{{ hash }}' type='checkbox' name='layer-toggle-{{ hash }}' value='network'>
```

Comment:
Network visibility toggle does not work. Hmm but not sure how important that is since that is what the user wants to see. Could just add the visibility prop to layers or remove the html if not needed.

Same thing with connected/disconnected buildings toggle. I think we can just remove them if it's not important so that we can reduce the complexity of the code.

Reply:
ah yes. i probably gutted the corresponding js code but didn't remove the html. on it.

Reply:
@reyery better now? I just removed all the layer toggles... kiss principle...